mwdb/web/src/types/types.ts

```diff
@@ -275,3 +281,19 @@ export type CreateUser = {
     feed_quality: FeedQuality;
     send_email: boolean;
 };
+
+export type ObjectOrConfigOrBlobData = ObjectData | ConfigData | BlobData;
+
+export type RepationItem = {
```

Comment:
To be changed in references as well
```suggestion
export type RelationItem = {
```